### PR TITLE
Seleniumcore

### PR DESCRIPTION
**Pull request by rak041988**
dfkhf
sdkjfhsdf
dskfhksdfh
fdfhjkhdf

### CODE REVIEW

File: src/test/java/com/selenium/mavenit/Hooks.java [hunk 1]
```diff
@@ -14,7 +14,7 @@ public void setUp(){
         driverManager.applyImpcitWait();
     }
 
-    @After
+   // @After
     public void tearDown(){
```
Comment (rak041988) [inline src/test/java/com/selenium/mavenit/Hooks.java:18]: y this commented out

File: src/test/java/com/selenium/mavenit/page_object/HomePage.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+public class HomePage extends DriverManager {
+
+    public String getCurrentUrl(){
+       return driver.getCurrentUrl();
+    }
+
+    public void doSearch(String item){
+        driver.findElement(By.cssSelector("input[data-test='search-input']")).sendKeys(item);
+        driver.findElement(By.cssSelector("input[data-test='search-input']")).sendKeys(Keys.ENTER);
+
```
Comment (rak041988) [inline src/test/java/com/selenium/mavenit/page_object/HomePage.java]: empty line

File: src/test/java/com/selenium/mavenit/step_def/FilterSteps.java [hunk 3]
```diff
@@ -1,21 +1,33 @@
...
-    public void i_search_for_product(String arg1) {
+    public void i_search_for_product(String searchTerm) {
+        homPage.doSearch(searchTerm);
     }
 
     @When("^I apply filter review \"([^\"]*)\"$")
-    public void i_apply_filter_review(String arg1) {
+    public void i_apply_filter_review(String filter) {
+        resultsPage.selectFilter(filter);
     }
```
Comment (rak041988) [inline src/test/java/com/selenium/mavenit/step_def/FilterSteps.java:23]: Hey no Assert? Test invalid

File: src/test/java/com/selenium/mavenit/Hooks.java (hunk 1 above)
Comment (rak041988) [reply src/test/java/com/selenium/mavenit/Hooks.java:18]: will be enabling back

File: src/test/java/com/selenium/mavenit/page_object/HomePage.java (hunk 2 above)
Comment (rak041988) [reply src/test/java/com/selenium/mavenit/page_object/HomePage.java]: done

File: src/test/java/com/selenium/mavenit/Hooks.java [hunk 4]
```diff
@@ -14,7 +14,7 @@ public void setUp(){
         driverManager.applyImpcitWait();
```
Comment (maninder1438) [inline src/test/java/com/selenium/mavenit/Hooks.java:14]: klgjnvkldfgdfgklm

Comment (rak041988) [reply src/test/java/com/selenium/mavenit/Hooks.java:14]: done

File: src/test/java/com/selenium/mavenit/step_def/FilterSteps.java (hunk 3 above)
Comment (mmustafa7) [reply src/test/java/com/selenium/mavenit/step_def/FilterSteps.java:23]: ok

**Review by maninder1438 — APPROVED** [review]

**Pull request merged**